pkg/uwb-dw1000/hal/uwb_dw1000.c

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (C) 2020 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_uwb_dw1000
+ * @{
+ *
+ * @file
+ * @brief       Glue code for running uwb-core for RIOT
+ * *
```

Comment:
author is missing, same in other files

Reply:
Yeah realized its missing from a lot of them